app/views/pages/items.js

```diff
@@ -70,6 +70,13 @@ let ItemsViewController = React.createClass({
     }
   },
 
+  componentWillReceiveProps: function(nextProps) {
+    if (this.getParams().id !== this.state.product.id) {
+      ProductActions.init(this.getParams().id);
```

Comment:
yeah the filters get flushed on product initialization

I'll have to re-think the filters sidebar